model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -242,7 +242,7 @@ def get_records(self):
         last perf_analyzer run
         """
 
-        if self._perf_records:
+        if self._output:
```

Comment:
How does this change the behaviour? Are we guaranteed that when there is output, there are records?

Reply:
There is no guarantee, but because of #189 change, the `analyze` part will gracefully handle no records output. If this is a concert, we can revert it back to checking `self._perf_records`.

Reply:
Reverted.